### ISSUE TITLE
ocamldebug and ocamldoc crash

### ISSUE THREAD

**Issue by peterjc** [conda-forge/ocaml-feedstock#3]
See logs on #2,

https://circleci.com/gh/conda-forge/ocaml-feedstock/4?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
Linux CircleCI,

```
$ ocamldebug -version
Fatal error: exception Failure("input_value: unknown custom block identifier")
```

https://travis-ci.org/conda-forge/ocaml-feedstock/builds/363182681?utm_source=github_status&utm_medium=notification
macOS, TravisCI,

```
$ ocamldebug -version
/Users/travis/miniconda3/conda-bld/ocaml_1523031087918/test_tmp/run_test.sh: line 7: 33648 Segmentation fault: 11  ocamldebug -version
```

https://travis-ci.org/conda-forge/ocaml-feedstock/builds/363190382?utm_source=github_status&utm_medium=notification
macOS, TravisCI,

```
$ ocamldoc -version
/Users/travis/miniconda3/conda-bld/ocaml_1523032262217/test_tmp/run_test.sh: line 8: 33653 Segmentation fault: 11  ocamldoc -version
```

**Comment by peterjc** [comment]
Logged upstream as https://caml.inria.fr/mantis/view.php?id=7771

**Comment by peterjc** [comment]
@isuruf I don't suppose all your recent cleanup work happened to fix this issue as a side effect, did it? In anycase, thank you.

P.S. Do you want to be listed as a maintainer of this recipe?

**Comment by isuruf** [comment]
I cleaned up the recipe a bit to fix some segfaults and fix #15. Hope you don't mind.
I don't use ocaml myself so I don't see myself maintaining this recipe. I cleaned up the recipe to help another developer who wanted ocaml for another conda package.

**Comment by peterjc** [comment]
Thank you for solving this @isuruf! Funnily enough the only reason I packaged OCaml in the first place was as a build time dependency for another tool that I wanted in conda (hevea).
